docs/overview/getting-started/avalanche-consensus.md

```diff
@@ -95,8 +129,9 @@ Random changes in preference, caused by random sampling, cause a network prefere
 which begets more network preference for that choice until it becomes irreversible and then the
 nodes can decide.
 
-:::info For a great visualization, check out [this
```

Comment:
Current bug in the consensus page, this makes the entire page below this in a blue info bubble.